CHANGELOG.md

```diff
@@ -4,6 +4,114 @@ All notable changes to this project will be documented in this file.
+      releases) can set a new `OPENSHIFT_INSTALL_OS_IMAGE_OVERRIDE`
+      environment variable.
+    - Per-platform network properties have been consolidated into the
+      platform-agnostic `machineCIDR` from which node IP addresses are
+      assigned.
+    - On libvirt, all machine-pool configuration has been removed, as
+      no remaining properties were supported by the libvirt
+      cluster-API provider.
+
+- `install-config.yaml` read during [staged
+  installs](docs/user/overview.md#multiple-invocations) will now have
+  installer-defaults applied for missing properties.  This allows you
+  to set only the properties you are interested in overriding, and
+  allow the installer to manage the remaining properties.
+- `create ignition-configs` now also writes the admin kubeconfig to
+  the asset directory, to support bring-your-own-infrastructure use
+  cases.
+- The bootstrap node now [serves
+  journals](docs/user/troubleshooting.md#troubleshooting-the-bootstrap-node)
+  for easier troubleshooting.
+- The validity for the initial kubelet TLS certificate has been
+  increased from one hour to 24 hours, to give
+  bring-your-own-infrastructure users longer to manually distribute
+  the certificate before it expires.
+- The key for the root certificate authority is no longer pushed into
+  the cluster (not even to the bootstrap node).
+- Machine(set)s generated by the installer now use `providerSpec`
+  instead of the deprecated `providerConfig`.
+- On AWS, the load balancers now use HTTPS health checks to reduce log
+  noise like:
+
+        http: TLS handshake error from 10.0.20.86:28372: EOF
+
+- On AWS, IAM roles are now tagged with the usual resource tags
+  (`openshiftClusterID`, etc.).  Some other resources have had their
+  tags updated to match those conventions (e.g. the internal Route 53
+  hosted zone was updated from `KubernetesCluster` to
+  `kubernetes.io/cluster/{name}: owned`).
+- The OpenStack platform has been removed from the install-config
+  wizard while it remains experimental.  It is still available for
+  users who supply their own `install-config.yaml`.
+- On OpenStack, the service VP now respects any SSH key specified in
+  the install configuration.
+- On OpenStack, a developer-only internal DNS server has been removed,
+  so users need to configure additional records for the existing
+  external DNS zone.
+- On OpenStack, Neutron trunk ports are now used for VM network
+  interfaces if Neutron supports them to support future Kuryr
+  integration.
+- On OpenStack, masters and workers have been consolidated in a single
+  subnet to simplify the deployment.
+- On OpenStack, the Ignition security group now only allows internal
+  connections, and no longer allows connections from outside the
+  cluster network.
+- On OpenStack, the machine(set) templates have been updated to set
+  `cloudName` and some other properties.
+- On libvirt, `destroy cluster` is not more robust in the face of
```

Comment:
s/not/now/

Reply:
> s/not/now/

Thanks :).  Fixed with 2cbe0f8 -> 997f778.  Can you drop a fresh `/lgtm`?